### ISSUE TITLE
move curriculum content to github

### ISSUE THREAD

**Issue by mikelmaron** [hotosm/hotosm#83]

**Comment by russdeffner** [comment]
I think this is related to #77?

**Comment by mikelmaron** [comment]
Not entirely. #77 is about setting up GitHub tickets to track actions needs on the curriculum. This ticket is about finding a flexible home for the content.

Does moodle have an export function?

**Comment by russdeffner** [comment]
I am not entirely sure off-the-top of my head, but it's got a whole bunch of stuff I don't know about. I think before we 'export' or try to make an editable source, it's needs a 'once-over'.

**Comment by dalekunce** [comment]
+1 for @mikelmaron suggestion. To give it the once over lets port everything here. It will make collaborating a lot easier with PRs and such.

**Comment by russdeffner** [comment]
Ok, I'm making some minor corrections today; will hold off on anything 'major'.

**Comment by smit1678** [comment]
@russdeffner @mikelmaron @dalekunce Is there interest to pick up this conversation again? Was this discussed in the Training or Activation working groups when this was brought up originally?

**Comment by smit1678** [comment]
It looks like it was picked up in https://github.com/hotosm/tech/issues/89

**Comment by russdeffner** [comment]
Yes, we can probably close this one.  Porting content to gitbooks is on hold at the moment, but I have been doing some direct updated of courses as necessary to keep everything complete-able.  Came up in last Training WG and will probably continue there until AWG is restarted.

**Comment by smit1678** [comment]
Ok thanks @russdeffner, was there a specific reason for putting this on hold or just the time and ability to do it?  

I'll throw some notes up for the next Training WG meeting. I'd like to talk through some other documentation needs that fit under both the Tech and Training WG and so it'd be good to discuss and coordinate.

**Comment by mikelmaron** [comment]
@smit1678 I think it would be great to have consistent framework and style guidance across all HOT documentation projects. Moving to GitBooks for several pieces, like the activation curriculum, could be a solid move.

**Comment by russdeffner** [comment]
This is a duplicate of #89

**Comment by russdeffner** [comment]
Here is the link to the Gitbooks site we started copying content to: https://www.gitbook.com/book/bgirardot/hotosm-activation-curriculum/details

**Comment by russdeffner** [comment]
This has been done, https://github.com/russdeffner/ActivationCurriculum & https://russdeffner.gitbook.io/curriculumworkspace/ - next, need to clone it to the HOT github. Anything I need to know before requesting that?
